pkg/types/conf.go

```diff
@@ -418,36 +418,6 @@ func LoadNetConf(bytes []byte) (*NetConf, error) {
 	return netconf, nil
 }
 
-const (
-	// const block for multus-daemon configs
-
-	// DefaultMultusDaemonConfigFile is the Default path of the config file
-	DefaultMultusDaemonConfigFile = "/etc/cni/net.d/multus.d/daemon-config.json"
-	// DefaultMultusRunDir specifies default RunDir for multus
-	DefaultMultusRunDir           = "/run/multus/"
-)
-
-// LoadDaemonNetConf loads the configuration for the multus daemon
-func LoadDaemonNetConf(config []byte) (*ControllerNetConf, error) {
```

Comment:
My only concern with moving this from here is that this could be more difficult to find when you're making other configuration changes

Reply:
I move this function to 'pkg/server' because of the function scope. I would like to specify that this function is only for 'thick' plugin.

Currently we have 'thick' and 'thin' plugin, so put it in same place as 'thin' pluign, it seems to be easy, however, it may confuse to change the code or to troubleshoot the code because thin/thick both codes are in same packages.

This might be a trade-off, so there is not definitive answer for that...

Reply:
I think I'm good with this -- it's ok that it's server specific.